src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -34,6 +34,7 @@ interface CheckoutState {
   expressCheckoutPaymentMethods: ExpressCheckoutPaymentMethod[] | null
   steps: CheckoutStep[]
   activeFulfillmentDetailsTab: FulfillmentDetailsTab | null
+  confirmationToken: any // Do we need a type for this object?
```

Comment:
It might be fine for now. I'm thinking eventually we might want to encapsulate it in its own interface, for both new and saved payment methods. Maybe something like `selectedPaymentMethod`.